docs-2.0/nebula-algorithm.md

```diff
@@ -122,6 +135,8 @@ Nebula Algorithm实现图计算的流程如下：
       # 数据源。Nebula Graph作为图计算的数据源时，nebula.read的配置才生效。
       read: {
           # 所有Meta服务的IP地址和端口，多个地址用英文逗号（,）分隔。格式: "ip1:port1,ip2:port2"。
+          # 使用docker-compose部署，端口需要填写docker-compose映射到外部的端口
+          # 可以用`docker-compose ps`查看
```

Comment:
Fix in the next PR plz. Some of the sentences end with a period while others do not. Correct it.